docs/contributing.md

```diff
@@ -72,36 +72,19 @@ you through every detail!
   Always use a `feature` branch. It's good practice to avoid
   work on the ``main`` branch of any repository.
 
-4.  We use [Poetry](https://python-poetry.org/) for packaging and dependency management, and project requirements are in       ``pyproject.toml``. We suggest using a [virtual environment](https://docs.python-guide.org/dev/virtualenvs/) for
-development. For those using Apple Silicon chips, we advise using [Conda miniforge](https://github.com/conda-forge/miniforge). Once the virtual environment is activated, run:
+4.  We use [Hatch](https://hatch.pypa.io/latest/) for packaging and dependency management. Project requirements are in ``pyproject.toml``. To install GPJax into a Hatch virtual environment, run:
 
   ```bash
-  $ poetry install
+  $ hatch env create
   ```
 
   At this point we recommend you check your installation passes the supplied unit tests:
 
   ```bash
-  $ poetry run pytest
+  $ hatch run dev:all-tests
   ```
 
-5.  Install the pre-commit hooks.
```

Comment:
No pre-commit hooks with Hatch?

Reply:
No. 1) A lot of people weren't using them anyway, and 2) I haven't figured out how to make them play nice with Hatch's environments. If it becomes a problem, then we can always reintroduce the pre-commit.